boards/FSM/Inc/App/App_AcceleratorPedals.h

```diff
@@ -2,6 +2,20 @@
 
 #include <stdbool.h>
 
+// TODO: Experimentally determine the encoder values for the primary and
+// secondary APPS when the accelerator is fully pressed #666
+#define PAPPS_ENCODER_FULLY_PRESSED_VALUE (2500U)
+#define SAPPS_ENCODER_FULLY_PRESSED_VALUE (1800U)
+
+// If encoder exceeds the following values, we reset it back to zero (usually
+// due to underflowing)
```

Comment:
Could we also add in the comments something like "E.g accelerator pedal pressed when FSM boots up"